src/drm/widevineConfig.ts

```diff
@@ -6,6 +6,11 @@ export interface WidevineConfig {
    * The DRM license acquisition URL.
    */
   licenseUrl: string;
+  /**
+   * A map containing the HTTP request headers, or null.
+   * Android only.
```

Comment:
The hole `WidevineConfig` is Android only I think. iOS does not support widevine AFAIK 🤔

Reply:
We actually have it as it can be used for Casting, but casting is not exposed yet.

Reply:
See: https://github.com/bitmovin-engineering/player-ios/blob/development/BitmovinPlayer/BitmovinPlayer/config/source/drm/WidevineConfig.swift

Reply:
Good point. I will move the comment to the class level in that case.

Reply:
[f9f07ab](https://github.com/bitmovin/bitmovin-player-react-native/pull/98/commits/f9f07ab8e5032594301e5ead3321998047845fef)